src/ontology/mondo-edit.obo

```diff
@@ -426792,8 +426798,7 @@ subset: orphanet_rare {source="Orphanet:636"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "neurofibromatosis" BROAD [NCIT:C3273]
-synonym: "Neurofibromatosis 1" EXACT [NORD:1502]
-synonym: "neurofibromatosis 1" EXACT [NCIT:C3273]
+synonym: "neurofibromatosis 1" EXACT [NCIT:C3273, NORD:1502]
```

Comment:
these changes were not requested on the ticket and are not mentioned in the commit message. Could you please explain these additional changes?
Thanks!